Marlin/src/module/stepper.h

```diff
@@ -303,10 +303,9 @@ class Stepper {
 
       #ifdef CPU_32_BIT
         // In case of high-performance processor, it is able to calculate in real-time
-        timer = (uint32_t)(HAL_STEPPER_TIMER_RATE) / step_rate;
-        if (timer < (HAL_STEPPER_TIMER_RATE / (STEP_DOUBLER_FREQUENCY * 2))) { // (STEP_DOUBLER_FREQUENCY * 2 kHz - this should never happen)
-          timer = (HAL_STEPPER_TIMER_RATE / (STEP_DOUBLER_FREQUENCY * 2));
-        }
+        constexpr uint32_t MIN_TIME_PER_STEP = (HAL_STEPPER_TIMER_RATE) / ((STEP_DOUBLER_FREQUENCY) * 2);
```

Comment:
Defining `MIN_TIME_PER_STEP` as a `constexpr` breaks the LPC1768 build with the following error:

```
Marlin/src/module/stepper.h: In static member function 'static uint32_t Stepper::calc_timer(uint32_t)':
Marlin/src/module/stepper.h:306:104: error: the value of 'SystemCoreClock' is not usable in a constant expression
constexpr uint32_t MIN_TIME_PER_STEP = (HAL_STEPPER_TIMER_RATE) / ((STEP_DOUBLER_FREQUENCY) * 2);
^
```

Changing `constexpr` to just `const` works.

Reply:
@tcm0116 @thinkyhead 
I can confirm that this breaks the LPC1768 build for my Re-ARM and that changing constexpr to just const works.